src/components/swap/SwapSkeleton.tsx

```diff
@@ -0,0 +1,103 @@
+import { ArrowContainer } from 'pages/Swap'
+import { ArrowDown } from 'react-feather'
+import styled, { useTheme } from 'styled-components/macro'
+
+import { ArrowWrapper } from './styleds'
+
+const LoadingWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  padding: 8px;
+  border: ${({ theme }) => `1px solid ${theme.backgroundOutline}`};
+  border-radius: 1rem;
+`
+
+const Blob = styled.div<{ height: string; width: string; radius?: number; marginTop?: number }>`
+  background-color: ${({ theme }) => theme.backgroundModule};
+  border-radius: ${({ radius }) => (radius ?? 0.25) + 'rem'};
```

Comment:
it seems like we settled on `rem` - is there anywhere in readme/notion we can point people to as the "why" and some migration tips

Reply:
I still use pixels in my PR 😆